services/management/ia/templates_service.go

```diff
@@ -84,22 +82,23 @@ type TemplatesService struct {
 }
 
 // NewTemplatesService creates a new TemplatesService.
-func NewTemplatesService(db *reform.DB) (*TemplatesService, error) {
+func NewTemplatesService(db *reform.DB, portalClient *platform.Client) (*TemplatesService, error) {
 	l := logrus.WithField("component", "management/ia/templates")
 
 	err := dir.CreateDataDir(templatesDir, "pmm", "pmm", dirPerm)
 	if err != nil {
 		l.Error(err)
 	}
 
-	host, err := envvars.GetSAASHost()
+	host, err := envvars.GetPlatformAddress()
 	if err != nil {
 		return nil, err
 	}
 
 	s := &TemplatesService{
 		db:                db,
 		l:                 l,
+		portalClient:      portalClient,
 		userTemplatesPath: templatesDir,
 		host:              host,
```

Comment:
looks like `host` is not used anywhere else.

Reply:
the same for `publicKeys`

Reply:
Removed